R/estimate_cox.R

```diff
@@ -0,0 +1,181 @@
+#' @title Wrapper for Cox Proportional Hazard analysis
+#'
+#' @description This function is a wrapper around \code{survival::coxph} to fit a Cox Proportional Hazards Regression model, assuming right-censored data.
+#'    The function expects that the data has been filtered on the parameter (PARAM/PARAMCD) of interest.
+#'    The result is an object of class \code{summary.coxph} which can be used in downstream functions and methods that rely on the \code{summary.coxph} class.
+#'
+#' @param data The name of the dataset for the Cox model is based on the Analysis Data Model (ADaM) principles. The dataset is expected to have
+#'    one record per subject per analysis parameter. Rows in which the analysis variable (AVAL) or the sensor variable (CNSR) contain NA, are removed during analysis.
+#' @param equation Character vector, representing the covariates used at the Cox model. When NULL, the baseline (null) model is fit.
```

Comment:
#' @param equation Character vector, representing the formula to be estimated with the Cox model. When NULL, the baseline (null) model is fit.

Reply:
Done